Automattic/wp-calypso
# Quick links: list Jetpack functionalities for Business or Commerce sites

Related to https://github.com/Automattic/dotcom-forge/issues/2665

## Proposed Changes

* List Jetpack features in Quick Links to help promote plan. Changes have been made for devs and non-devs with Business or eEcommerce sites. Simple sites remained unchanged. 
* Speed up your site -> `https://${ siteSlug }/wp-admin/admin.php?page=jetpack-boost` 
* Restore a back -> `/backup/${ siteSlug }`

| Simple | Devs with Atomic | Non-devs with Atomic |
| ------------- | ------------- | ------------- |
| <img width="307" alt="Screenshot 2566-06-26 at 16 34 41" src="https://github.com/Automattic/wp-calypso/assets/6851384/d809ce07-26e6-48eb-866d-148e74fc991f"> | ![image](https://github.com/Automattic/wp-calypso/assets/6851384/50e4e2d6-1de3-47d3-8a6b-e847591203c8)  | ![image](https://github.com/Automattic/wp-calypso/assets/6851384/ecc339bc-5a38-4866-94a6-63d66aa05194)  |



## Testing Instructions



* Try visiting `/home` for 
    - Atomic site with Dev account
        * with Boost
        * without Boost 
    - Atomic site without Dev account
        * with Boost
        * without Boost 
    - Simple site
* I had to add some extra styling for the Jetpack logo - can you think of a better way to render this icon?
* Are the links pointing to where you expect them to?
* Should we show these links even if the Jetpack user hasn't got Boost and Backup?
* Try it with a RTL language or with a different theme

![image](https://github.com/Automattic/wp-calypso/assets/6851384/6426f879-aacc-41a2-9a1e-4c6571ca397f)


## Pre-merge Checklist
